SourcingTests/FetchedResultsDataProviderTests.swift

```diff
@@ -201,6 +216,51 @@ class FetchedResultsDataProviderTests: XCTestCase {
         XCTAssert(didUpdateDataSource)
     }
     
+    func testProcessUpdatesNumberForMoveChange() {
```

Comment:
we should make sure that we test there that one move operation is decided in a move and and update to the newIndexPath

Reply:
right, will fix it today

Reply:
Thanks @pkalinowski.